include/polycap-description.h

```diff
@@ -25,9 +33,27 @@ extern "C" {
 #endif
 
 struct _polycap_description;
+/** Struct containing information about a polycapillary description such as shape and composition
+ *
+ * When this struct is no longer required, it is the user's responsability to free the memory using polycap_description_free().
+ */
 typedef struct _polycap_description                 polycap_description;
 
-// get a new polycap_description by providing all its properties... perhaps a simpler variant of this function could be defined that would only set the most important parameters and use defaults for the others??
+//TODO: figure out what sig_wave and corr_length actually represent and implement in code, or just get rid of them as parameters...
+/** Creates a new polycap_description by providing all its properties.
+ *
+ * \param profile polycap_profile containing outer polycapillary and single capillary shape coordinates
+ * \param sig_rough Surface rougness of the capillaries [$\AA$]
```

Comment:
Ik meende gelezen te hebben dat het Latex commands aankon, maar \AA voor Angstrom werkt niet dan?

Reply:
Probeer eens de html code voor de [angstrom](https://www.toptal.com/designers/htmlarrows/symbols/angstrom-sign/):

```html
&#8491;
```

Reply:
werkt niet :( Heb ook de hexadecimale al geprobeerd, geeft ook niks. Zal maar gewoon voluit Angstrom schrijven :P

Reply:
Hmmm... die punt komma niet vergeten?

Probeer anders ook eens gewoon een angstrom in de tekst te krijgen via de editor zelf door een special character in te voegen met de terminal